app/src/main/java/sdp/moneyrun/MenuActivity.java

```diff
@@ -280,5 +288,29 @@ public PopupWindow onButtonShowPopupWindowClick(View view, Boolean focusable, in
 //        }
 //        //TODO: put player in the database with playerId as primary key
 //    }
-    
+
+    public void onButtonShowQuestionPopupWindowClick(View view, Boolean focusable, int layoutId, Riddle riddle) {
```

Comment:
I assume this happened when you resolved conflicts, but this should not be here.

Reply:
fixed